### PR TITLE
Wqp 1326 - Migrate NWIS Schema to Postgres

### PR DESCRIPTION
**Pull request by mbucknell**
Removed the database links since these will likely be done by changing the ETL to query the database, transforming the data, and then loading rather than using a direct database link.

The etlHelper package was changed to a function.

Used text rather than character varying for large fields.

### CODE REVIEW

File: liquibase/changeLogs/wqp/preload/tables/altitudeMethod/altitudeMethod.sql [hunk 1]
```diff
@@ -0,0 +1,9 @@
+create unlogged table if not exists ${NWIS_WS_STAR_SCHEMA_NAME}.altitude_method
+(code                           character varying (25) primary key
```
Comment (dsteinich) [inline liquibase/changeLogs/wqp/preload/tables/altitudeMethod/altitudeMethod.sql]: I don't like to leave constraint naming to the database. At least this engine appears to make reasonable names...
If we don't name them all the time, we should change the ones we do name to match the generated names (*_pkey vs *_pk)

File: liquibase/changeLogs/wqp/preload/tables/intQwResult/intQwResult.sql [hunk 2]
```diff
@@ -0,0 +1,47 @@
+create unlogged table if not exists ${NWIS_WS_STAR_SCHEMA_NAME}.int_qw_result
+(nwis_host_nm        character varying (42)
+,db_no               character varying (6)
+,record_no           character varying (24)
+,parm_cd             character varying (15)
+,meth_cd             character varying (15)
+,result_va           double precision
```
Comment (dsteinich) [inline liquibase/changeLogs/wqp/preload/tables/intQwResult/intQwResult.sql]: I think we should use decimal - double precision is "inexact"... https://www.postgresql.org/docs/current/datatype-numeric.html#DATATYPE-FLOAT

File: liquibase/changeLogs/wqp/preload/tables/parmMeth/parmMeth.sql [hunk 3]
```diff
@@ -0,0 +1,12 @@
+create unlogged table if not exists ${NWIS_WS_STAR_SCHEMA_NAME}.parm_meth
+(parm_cd                        character varying (5)    constraint nn_parm_meth_parm not null
```
Comment (dsteinich) [inline liquibase/changeLogs/wqp/preload/tables/parmMeth/parmMeth.sql]: While it doesn't flag this as an error, postgreSQL ignores naming constraints this way.

File: liquibase/changeLogs/wqp/preload/functions/etlHelper/determineProjectId.sql [hunk 4]
```diff
@@ -1,21 +1,19 @@
...
-        rtn varchar2(4000 char);
+create or replace function ${NWIS_WS_STAR_SCHEMA_NAME}.determine_project_id
+     (p_site_no           in ${NWIS_WS_STAR_SCHEMA_NAME}.nawqa_sites.site_no%type,
+      p_50280_value       in ${NWIS_WS_STAR_SCHEMA_NAME}.qw_result.result_unrnd_va%type,
+      p_71999_value       in ${NWIS_WS_STAR_SCHEMA_NAME}.qw_result.result_unrnd_va%type,
+      p_sample_start_date in ${NWIS_WS_STAR_SCHEMA_NAME}.qw_sample.sample_start_dt%type,
+      p_project_cd        in ${NWIS_WS_STAR_SCHEMA_NAME}.qw_sample.project_cd%type)
+        returns text as '
+    declare
+        nawqa text := ''National Water Quality Assessment Program (NAWQA)'';
```
[the remark this answers is not in the document]
Comment (mbucknell) [reply liquibase/changeLogs/wqp/preload/functions/etlHelper/determineProjectId.sql]: I tried it that way originally. I'll try again. Maybe I had something else wrong.

File: liquibase/changeLogs/wqp/preload/tables/parmMeth/parmMeth.sql (hunk 3 above)
Comment (mbucknell) [reply liquibase/changeLogs/wqp/preload/tables/parmMeth/parmMeth.sql]: Left it in inadvertently. I will take it out.

File: .env [hunk 5]
```diff
@@ -0,0 +1,10 @@
+POSTGRES_PASSWORD=postgres
```
[the remark this answers is not in the document]
Comment (mbucknell) [reply .env]: Shoot! I thought I removed it from the commit. I'll do so now.

File: liquibase/changeLogs/wqp/preload/functions/etlHelper/determineProjectId.sql (hunk 4 above)
[the remark this answers is not in the document]
Comment (mbucknell) [reply liquibase/changeLogs/wqp/preload/functions/etlHelper/determineProjectId.sql]: When I run the liquibase script, I get ```Unexpected error running Liquibase: Unterminated dollar quote started at position 515 in SQL create or replace function```.  I don't see any differences with NLDI example.

**Pull request merged**